server.js

```diff
@@ -63,36 +82,49 @@ app.use(function (req, res, next) {
   next();
 });
 
+// Redirect root to /docs when in promo mode.
+if (promoMode == 'true'){
+  console.log('Kit running in promo mode');
+  app.get('/', function (req, res) {
+    res.redirect('/docs');
+  });
+}
+
 // routes (found in app/routes.js)
 if (typeof(routes) != "function"){
   console.log(routes.bind);
-  console.log("Warning: the use of bind in routes is deprecated - please check the prototype kit documentation for writing routes.")
+  console.log("Warning: the use of bind in routes is deprecated - please check the prototype kit documentation for writing routes.");
   routes.bind(app);
 } else {
   app.use("/", routes);
 }
 
+// Downloads returns a url to the zip of the latest release on github
+app.get('/prototype-admin/download-latest', function (req, res) {
+  url = utils.getLatestRelease();
+  res.redirect(url);
+});
+
+if (useDocs == true){
+  // Create separate router for docs
+  app.use("/docs", docsApp);
+  // Docs under the /docs namespace
+  docsApp.use("/", docsRoutes);
+}
+
 // auto render any view that exists
+
+// App folder routes get priority
 app.get(/^\/([^.]+)$/, function (req, res) {
+  utils.matchRoutes(req, res);
+});
 
-  var path = (req.params[0]);
-
-  res.render(path, function(err, html) {
-    if (err) {
-      res.render(path + "/index", function(err2, html) {
-        if (err2) {
-          console.log(err);
-          res.status(404).send(err + "<br>" + err2);
-        } else {
-          res.end(html);
-        }
-      });
-    } else {
-      res.end(html);
-    }
+// Docs folder routes
+if (useDocs){
```

Comment:
comparison to a boolean should use === otherwise JavaScript can do weird things.